### PR TITLE
New Game! & Pokemon

### PR DESCRIPTION
**Pull request by beschoenen**

### CODE REVIEW

**Comment by beschoenen** [comment]
Anyone @SickRage/collaborators?

File: scene_exceptions/scene_exceptions.json [hunk 1]
```diff
@@ -507,7 +507,72 @@
         },
         "76703": {
             "-1": [
-                "Pokemon"
+                "Pokemon",
+                "Pocket Monsters"
```
Comment (OmgImAlexis) [inline scene_exceptions/scene_exceptions.json]: Pocket Monsters tends to get the non English version.

Comment (beschoenen) [inline scene_exceptions/scene_exceptions.json]: That's what I want though, but I can see how that could be a problem. Should I remove the other references to it as well?

Comment (duramato) [inline scene_exceptions/scene_exceptions.json]: Leave it, `Pocket Monsters` is the original audio

Comment (OmgImAlexis) [inline scene_exceptions/scene_exceptions.json]: @duramato and now what happens if a lot of users get the original when they expect the English version?

Comment (duramato) [inline scene_exceptions/scene_exceptions.json]: They can use the ignored words feature

**Comment by OmgImAlexis** [comment]
@miigotu that shouldn't have been merged.

**Comment by miigotu** [comment]
why?

**Comment by OmgImAlexis** [comment]
Pocket monsters refers to the original japanese version where as Pokemon refers to the English version. This is going to cause users to get the wrong one.

**Comment by beschoenen** [comment]
This has been a long standing discussion on TheTVDB as well, and they've chosen for the Japanese seasons and episodes cause that's where it airs first.

**Comment by OmgImAlexis** [comment]
@beschoenen this has nothing todo with tvdb or ordering, etc. it's that users will now end up with japanese copies of the show instead of english ones as most copies of "Pocket monsters" don't have any english audio at all in them where as "Pokemon" is the opposite with only English audio.

**Comment by beschoenen** [comment]
What I mean is that if you download episode 18x10 for instance if won't be the correct (English) one cause it might be a different episode

**Comment by OmgImAlexis** [comment]
That's what [xem](http://thexem.de/) is for...

**Comment by duramato** [comment]
> That's what xem is for...

And if you don't want a certain dub that's what ignored words are for... Certain user preferences shouldn't impair overhaul download experience. Just like you want the English dub, **n** others want the original Japonese dub too.

**Pull request merged**